### PR TITLE
basic querying & mutation

### PR DESCRIPTION
**Pull request by PDhanrajnath**

### CODE REVIEW

File: Apollo-Server/index.js [hunk 1]
```diff
@@ -4,22 +4,22 @@ const { ApolloServer, gql } = require("apollo-server");
...
-  # This "Book" type defines the queryable fields for every book in our data source.
   type Book {
     title: String
     author: String
   }
-
-  # The "Query" type is special: it lists all of the available queries that
-  # clients can execute, along with the return type for each. In this
-  # case, the "books" query returns an array of zero or more Books (defined above).
+  type User {
```
Comment (rohithZE) [inline Apollo-Server/index.js:7]: create different files for each type of schema and each type of resolver.
You can merge all the schema into one.
Not scalable to have single file having all the resolvers and schema

Comment (PDhanrajnath) [reply Apollo-Server/index.js:7]: ok Rohith
I'll do that

**Comment by rohithZE** [comment]
Also do you need a separate sandbox?
You can directly run the apollo on playground
https://www.apollographql.com/docs/apollo-server/v2/testing/graphql-playground/#:~:text=GraphQL%20Playground%20is%20a%20graphical,the%20GUI%20to%20web%20browsers.

**Comment by PDhanrajnath** [comment]
Hi Rohith
I used user schema and performed create, delete & read operations
I'll try to separate schema, resolvers files tomorrow

File: Apollo-Server/index.js [hunk 2]
```diff
@@ -4,62 +4,58 @@ const { ApolloServer, gql } = require("apollo-server");
...
   }
   type Mutation {
-    addBook(title: String, author: String): Book
+    addUser(id: String, name: String): User
+    deleteUser(id: String): String
   }
 `;
 
-const books = [
+const users = [
```
Comment (rohithZE) [inline Apollo-Server/index.js:20]: create a test constants file and import it from there.

File: Apollo-Server/index.js [hunk 3]
```diff
@@ -4,62 +4,58 @@ const { ApolloServer, gql } = require("apollo-server");
...
-        title: args.title,
-        author: args.author,
+    addUser: (_parent, args, context) => {
+      const usr = {
+        id: args.id,
+        name: args.name,
       };
-      books.push(book);
+      users.push(usr);
       console.log(context.key);
```
Comment (rohithZE) [inline Apollo-Server/index.js:44]: console logs should never be pushed to dev

File: hackernews-react-apollo/src/components/LinkList.js [hunk 4]
```diff
@@ -1,26 +1,28 @@
...
-import Link from "./link";
+// import Link from "./link";
 
-const FEED_QUERY = gql`
+const GET_USER = gql`
   {
-    books {
-      title
-      author
+    user(id: "2") {
```
Comment (rohithZE) [inline hackernews-react-apollo/src/components/LinkList.js:7]: don't hardcode id
see how can you write queries with dynamic parameters.
It is straight forward

**Pull request closed without merging**